<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="mahis-MacBook-Pro.local" failures="2" tests="17" name="FailedTests" time="5.768" errors="0" timestamp="2024-07-01T09:10:06 IST">
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test1" time="2.106"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test2" time="3.443"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.026"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.02"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.017"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.017"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.02"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.017"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.018"/>
  <testcase classname="com.test.test_scenarios.TestScenario1" name="test3" time="0.019"/>
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo1" time="0.001"/>
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo2" time="0.001">
    <skipped/>
  </testcase> <!-- demo2 -->
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo2" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.test_scenarios.TestScenario2.demo2(TestScenario2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo3" time="0.001"/>
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo4" time="0.001">
    <skipped/>
  </testcase> <!-- demo4 -->
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo4" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.test.test_scenarios.TestScenario2.demo4(TestScenario2.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- demo4 -->
  <testcase classname="com.test.test_scenarios.TestScenario2" name="demo5" time="0.001"/>
</testsuite> <!-- FailedTests -->
